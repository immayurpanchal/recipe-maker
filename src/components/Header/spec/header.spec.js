import React from 'react';
import { render } from '@testing-library/react';
import Header from '../Header';

describe('Header component', () => {
	it('Render the component', () => {
		const { container } = render(<Header />);
		expect(container).toMatchSnapshot();
	});
});
