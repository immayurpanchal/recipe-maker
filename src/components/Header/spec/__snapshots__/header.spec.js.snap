// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Render the component 1`] = `
<div>
  <header
    class="header"
  >
    <img
      alt="Logo"
      class="header__logo"
      src="img/logo.png"
    />
    <form
      class="search"
    >
      <input
        class="search__field"
        placeholder="Search over 1,000,000 recipes..."
        type="text"
      />
      <button
        class="btn search__btn"
        type="submit"
      >
        <svg
          class="search__icon"
        >
          <use
            href="img/icons.svg#icon-magnifying-glass"
          />
        </svg>
        <span>
          Search
        </span>
      </button>
    </form>
    <div
      class="likes"
    >
      <div
        class="likes__field"
      >
        <svg
          class="likes__icon"
        >
          <use
            href="img/icons.svg#icon-heart"
          />
        </svg>
      </div>
      <div
        class="likes__panel"
      >
        <ul
          class="likes__list"
        />
      </div>
    </div>
  </header>
</div>
`;
